### PR TITLE
module #6 [Theory] Architecture advanced - MVC, 3-layered, gradle modules

### PR DESCRIPTION
**Pull request by AliaksandrLiulis**

### CODE REVIEW

File: stage #2.5/module #6 [Theory] Architecture advanced - MVC, 3-layered, gradle modules/#1. Layered Architecture/content.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<H1 style="color: green">Layered Architecture</h1>
+
+***
```
Comment (ShiniTada) [inline stage #2.5/module #6 [Theory] Architecture advanced - MVC, 3-layered, gradle modules/#1. Layered Architecture/content.md:3]: this *** create strange grey line. Please remove it

File: stage #2.5/module #1 Architecture advanced/#1. Layered Architecture/content.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+<H1 style="color: green">Layered Architecture</h1>
+
+***
+
+## Pattern Description
+
+Layered architecture patterns are n-tier patterns where the components are organized in horizontal layers and each layer has a specific role in the application (e.g., presentation logic or business logic). It means to be self-independent. All components should be interconnected, but not dependent on each other.
+
```
Comment (ShiniTada) [inline stage #2.5/module #1 Architecture advanced/#1. Layered Architecture/content.md:6]: Minor: "e.g.," - extra comma

File: stage #2.5/module #6 [Theory] Architecture advanced - MVC, 3-layered, gradle modules/#1. Layered Architecture/content.md (hunk 1 above)
Comment (AliaksandrLiulis) [reply stage #2.5/module #6 [Theory] Architecture advanced - MVC, 3-layered, gradle modules/#1. Layered Architecture/content.md]: But it strange. Can't see this locally.

File: stage #2.5/module #1 Architecture advanced/#1. Layered Architecture/content.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
```
Comment (ShiniTada) [inline stage #2.5/module #1 Architecture advanced/#1. Layered Architecture/content.md:33]: Сonsider splitting this paragraph into several smaller ones. The material is a little difficult to read in one paragraph

File: stage #2.5/module #1 Architecture advanced/#2. MVC/content.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+The concept of MVCs was first introduced by Trygve Reenskaug, who proposed it as a way to develop desktop application [GUIs](https://folk.universitetetioslo.no/trygver/themes/mvc/mvc-index.html).
+
+## Basic concepts
+### Separation of Concerns
+
+MVC design allows for Separation of Concerns - dividing the logic up between the 3 buckets, so that each bucket can act independently.
+The model, view, and controller don’t depend on each other. Why does this matter? Generally, software is worked on by teams - a team might have a designer, engineer, and database architect. Separation of concerns means each team member can work on their piece at the same time, because logic has been separated into buckets. Separation of concerns is also great for maitenance - developers can fix a bug in one piece of code, without having to check out the other pieces of code.
+
+### Loosely Coupled
```
Comment (ShiniTada) [inline stage #2.5/module #1 Architecture advanced/#2. MVC/content.md:26]: minor: maybe "Loose Coupling" suits better

**Comment by ShiniTada** [comment]
@AliaksandrLiulis Great job!! It's easy to read and understand. Thank you so much)

**Comment by ShiniTada** [comment]
We are also waiting for Katya's review

**Review by ShiniTada — APPROVED** [review]

**Review by Katerina-Charakhovich — APPROVED** [review]

**Pull request merged**